.changelog

```diff
@@ -4,6 +4,10 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)#Changelog_section_.28.25changelog.29
 
+- Updated CLM examples for Red Hat Enterprise Linux 9 (bsc#1226687)
+- Added VM based proxy installation in Installation and Upgrade Guide
+- Fixed PostgreSQL name entity
```

Comment:
Right now, I do not see any docs changes for these entries. Will they still come or are they added by mistake?

Reply:
Oops, thanks for spotting.  I resolved a cherry-pick conflict incompletely.  Now hopefully fixed.